table/sst_file_writer.cc

```diff
@@ -396,9 +396,9 @@ Status SstFileWriter::Open(const std::string& file_path, Temperature temp) {
       r->internal_comparator, &internal_tbl_prop_coll_factories,
       compression_type, compression_opts, cf_id, r->column_family_name,
       unknown_level, false /* is_bottommost */, TableFileCreationReason::kMisc,
-      0 /* oldest_key_time */, 0 /* file_creation_time */,
-      "SST Writer" /* db_id */, r->db_session_id, 0 /* target_file_size */,
-      r->next_file_number);
+      0 /* oldest_key_time */, kUnknownNewestKeyTime,
```

Comment:
It seems ingested file won't be TTL compacted. This is an existing behavior that we can fix as a follow up.